waspc/tools/ensure_todoapp_builds.sh

```diff
@@ -13,6 +13,9 @@ cd "$dir/../examples/todoApp"
 # Compile example app.
 cabal run wasp-cli build
 
+echo "Ensure the app has no IDE type errors"
+npx tsc --noEmit --skipLibCheck
```

Comment:
Building with `tsc` in the project root dir gives us the same IDE errors users would see.

Reply:
Just curios, why do we need to mention "IDE" here? I know they are visible in the IDE, but they are not really IDE specific, right? Aren't these TS errors and that is it -> the fact that they are shown by IDE sounds irrelevant. They could be shown by Wasp CLI also. I think we care about the fact that there are type errors from user's code in their Wasp project? So maybe let's call them that way then, if that is correct.

Reply:
> Aren't these TS errors and that is it

Yes, but also no. :smile: 
Due to how our dev setup works, the only place they'll see them is inside the IDE.

> They could be shown by Wasp CLI also.

Yes, but they aren't. They're only shown in the IDE (by design).

All that said, I agre with @Martinsos , IDE type errors is a little misleading.

I'd still mention the IDE though, mostly because user will report these errors as "I have errors in my IDE, but everything works". They'll see it as IDE-related because they don't know about all the crazy stuff we do :smile:  

Anyway, I suggest:
```
Ensure the user's code has no TypeScript errors (shown in the IDE).
```
A little verbose, but this way it's both:
- Correct.
- Easy to figure out what you need to look at when a user reports their IDE throwing errors.